format-common/src/main/java/io/cdap/plugin/format/input/PathTrackingInputFormatProvider.java

```diff
@@ -54,17 +56,42 @@ public Map<String, String> getInputFormatConfiguration() {
   }
 
   /**
-   * Perform validation on the provided configuration.
-   *
-   * Deprecated since 2.3.0. Use {@link PathTrackingInputFormatProvider#validate(FormatContext)} method instead.
-   */
+  * Perform validation on the provided configuration.
```

Comment:
these indentation changes shouldn't be done

Reply:
Done